requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+-r requirements.txt
```

Comment:
hey, but I have a question (maybe a stupid one). So these dependencies are needed to be installed (while developing) using `pip  install -r requirements-dev.txt` ?